modules/custom/openy_repeat/src/RepeatManager.php

```diff
@@ -72,29 +67,53 @@ class RepeatManager implements SessionInstanceManagerInterface {
    */
   protected $config;
 
+  /**
+   * The module handler service.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * Drupal\openy_moderation_wrapper\EntityModerationStatus definition.
+   *
+   * @var \Drupal\openy_moderation_wrapper\EntityModerationStatus
+   */
+  protected $moderationWrapper;
+
   /**
    * Constructor.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager, QueryFactory $entity_query, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
-    $this->entityQuery = $entity_query;
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory, ModuleHandlerInterface $module_handler, EntityModerationStatus $moderationWrapper) {
     $this->entityTypeManager = $entity_type_manager;
     $this->logger = $logger_factory->get(self::CHANNEL);
     $this->storage = $this->entityTypeManager->getStorage(self::STORAGE);
     $this->configFactory = $configFactory;
     $this->config = $this->configFactory->get('openy_repeat.settings');
+    $this->moduleHandler = $module_handler;
+    $this->moderationWrapper = $moderationWrapper;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
```

Comment:
@AleksandrPrykhodkoDev this service is defined in the YML file, so why do you use this approach here? It doesn't work, the service is broken now. Please define all dependencies in `openy_session_instance.services.yml` and update the original `SessionInstanceManager.php` service for that.